staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -175,6 +176,10 @@ type Config struct {
 	// if the client requests it via Accept-Encoding
 	EnableAPIResponseCompression bool
 
+	// MergedResourceConfig indicates which groupVersion enabled and its resources enabled/disabled.
+	// This is passed from command line.
+	MergedResourceConfig *serverstore.ResourceConfig
```

Comment:
document what nil does.  Most people shouldn't have to worry about this.

Reply:
make sense